.github/workflows/image-builder.yml

```
@@ -0,0 +1,49 @@
+# Build a docker image on push to develop
+# Also tag the image if commit is tagged as an official release
+# all releases and CI build an image
+
+# what happens if we have develop push AND release events happen...
+name: 'Image Builder'
+
+on:
+  pull_request:
+    branches:
+    - 'ci_test'
+  push:
+    branches:
+    - 'ci_test'
```

like this for testing, will become
```suggestion
  push:
    branches:
    - develop
```
to run image build on any push to `develop` branch